src/input_xml.F90

```diff
@@ -5946,13 +6018,14 @@ subroutine read_0K_elastic_scattering(nuc, libraries, library_dict)
         ! Read nuclide data from HDF5
         file_id = file_open(libraries(i_library) % path, 'r')
         group_id = open_group(file_id, name)
-        call resonant_nuc % from_hdf5(group_id, '0K')
+        call resonant_nuc % from_hdf5(group_id, temperature, &
+             TEMPERATURE_NEAREST, 1000.0_8)
```

Comment:
The problem right now is that we don't actually have 0K cross sections for testing purposes, so if the tolerance were small, the test would just fail because it wouldn't have any data to load. This allows it to load the lowest temperature and use that in place of the 0K cross section (what we were effectively doing before by assigning the 294K data to `nuclide_0K`).